packages/jest-cli/src/cli/index.ts

```diff
@@ -48,17 +41,20 @@ export async function run(maybeArgv?: Argv, project?: Path) {
   }
 }
 
-export const buildArgv = (maybeArgv: ?Argv) => {
+export const buildArgv = (maybeArgv?: Array<string>): Config.Argv => {
```

Comment:
type here was wrong. The test added in #7707 caught it when I tweaked the code to not give type errors 🙂